css/index.css

```diff
@@ -143,4 +145,129 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+/* MY CODE  */
+
+nav div img {
+    margin-top: 20px;
+}
+nav {
+    display: flex;
+    justify-content: space-around;
+    margin: 10px 0 0 0;
+}
+
+.nav {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+}
+
+header nav a {
+    text-decoration: none;
+    font-size: 1.5rem;
+    color: black;
+    
+}
+
+
+
+a:hover {
+    color: orange;
+}
+
+#home-image {
+    display: flex;
+    justify-content: space-around;
+    margin: 30px 0 0 0;
+}
+
```

Comment:
Nice work using parent/child relationships to style the boxes.  You only changed the color to each box.